### PR TITLE
Snow day

### PR DESCRIPTION
**Pull request by ellezv**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""The setup for my katas modules."""
+
+from setuptools import setup
+
+setup(
+    name="code katas",
+    description="An implmentation of various katas from codewars.com",
```
Comment (nhuntwalker) [inline setup.py:7]: "implmentation = implementation"

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    version=0.1,
+    author="Maelle Vance",
+    author_email="maellevance@gmail.com",
+    license="MIT",
+    py_modules=['sum_positives', 'add_length', 'billboard', 'opposites', 'is_vowel', 'get_average', 'title_case', 'jaden_case', 'simple_pig_latin', 'lightsaber'],
+    package_dir={'': 'src'},
+    install_requires=['numpy', 'tox'],
```
Comment (nhuntwalker) [inline setup.py:14]: Interesting that you required tox instead of just putting it in the test extras

File: src/billboard.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"""Implementation of Name on Billboard Kata on CodeWars"""
+from numpy import multiply
+
+
+def billboard(name, price=30):
+    """Return cost of printing name on billboard."""
+    return multiply(len(name), price)
```
Comment (nhuntwalker) [inline src/billboard.py:7]: This is what you needed numpy's "multiply" for? You couldn't just use the `*` operator?

File: src/jaden_case.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+"""Implementation of Jade Casing String Kata from Code Wars."""
+
+
+def jaden_case(str_):
+    """Return a capitalized string."""
+    jaden_arr = [word.capitalize() for word in str_.split()]
+    return ' '.join(jaden_arr)
+"""Test for jaden_case module"""
```
Comment (nhuntwalker) [inline src/jaden_case.py:8]: This shouldn't be here

File: src/lightsaber.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+"""Implementation of the How Many Lightsabers Do You Own Kata on Code Wars."""
+
+
+def num_lightsaber(name=""):
+    """Return the name of lightsabers owned by the person."""
+    if name == "Zach":
+        return 18
+    return 0
```
Comment (nhuntwalker) [inline src/lightsaber.py:8]:
```python
def num_lightsaber(name=""):
    return 18 if name == "Zach" else 0
```

File: src/simple_pig_latin.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+def pig_it(text):
+    """Return a string a piglatin string."""
+    lst = []
+    for word in text.split():
+        if word == '!' or word == '?':
+            lst.append(word)
+        else:
+            word = word[1:] + word[0] + 'ay'
+            lst.append(word)
```
Comment (nhuntwalker) [inline src/simple_pig_latin.py:12]: you should only need one `lst.append(word)` in here.
```python
if word not in ["!", "?"]:
    word = word[1:] + word[0] + "ay"
lst.append(word)

File: src/tests/test_is_vowel.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    [[107, 99, 110, 107, "e", 106, 112, 102], [107, 99, 110, 107, 101, 106, 112, 102]],
+    [[100, 100, 116, "i", "u", 121], [100, 100, 116, 105, 117, 121]],
+]
+
+
+@pytest.mark.parametrize("result, input", NUM_TABLE)
+def test_is_vow(result, input):
+    """Test the is_vow function."""
+    from is_vowel import is_vow
+    assert is_vow(input) == result
```
Comment (nhuntwalker) [inline src/tests/test_is_vowel.py:15]: `input` is a python built-in. You should use a different variable name

File: src/billboard.py (hunk 3 above)
Comment (ellezv) [reply src/billboard.py:7]: The kata specified to not use the `*` operator so I used numpy instead. I think this is not what the kata author had in mind though.

File: src/simple_pig_latin.py (hunk 6 above)
Comment (ellezv) [reply src/simple_pig_latin.py:12]: Yes I saw that in other answers. Definitely smarter.

File: src/tests/test_is_vowel.py (hunk 7 above)
Comment (ellezv) [reply src/tests/test_is_vowel.py:15]: fixed!

File: src/jaden_case.py (hunk 4 above)
Comment (ellezv) [reply src/jaden_case.py:8]: fixed

File: setup.py (hunk 2 above)
Comment (ellezv) [reply setup.py:14]: tox now an extra.

File: setup.py (hunk 1 above)
Comment (ellezv) [reply setup.py:7]: fixed

**Pull request merged**